### DISCUSSION TITLE
Installation issue on Orangepi zero2 - unable to access settings page

### DISCUSSION THREAD

**Discussion by joshindc** [mcguirepr89/BirdNET-Pi#715] [General]
I was hoping I could get some help getting my installation up and running. I've purchased an OrangePi Zero 2 (w/1GB ram) as of course raspberry pis are sold out everywhere, hoping to get birdNet-Pi up and running. I get through the install script and am able to see the web interface when I navigate to the ip address in a browser, but I'm not able to change any settings. When I click on the tools tab, it (usually) prompts for username and password, and I enter birdnet without a password and it shows me the list of options (Settings, System Info, System Controls, etc). When I click on most of these links I'm able to access the page without issue, but when I click on settings the login request pops up again and birdnet without a password does not work. I suspect this is not my only issue, but it's difficult to troubleshoot further without being able to look at the settings!

Here's my install process so far:
I downloaded and used etcher to copy the Orangepizero2_3.0.6_debian_bullseye_server_linux5.16.17.7z image from the orangepi website (armbian images would all bootloop after a restart).
used ssh over ethernet to gain access as there was no way to change the config files like on a raspberry pi image 
Then I apt-get update and upgraded 
I changed the name of the non-root user to birder (I don't think that's needed but I wanted to try anything I could think of)
Ran: apt-get install python3-venv
That gave me an error and suggested fixing with:
apt fix-broken install
Then I reran apt-get install python3-venv which completed this time
Then I ran the curl command to install birdnet. The script completes and reboots the machine, and as I said I'm able to see the webpage but not change any settings.

Any suggestions on things to try? I realize I'm outside the expected use-case with different hardware, but I feel like I'm close and am hitting a brick wall.

Thanks in advance for your help,
Josh

**Comment by ehpersonal38** [comment]
Hi,

Could you please follow these steps to enable php error logging: https://github.com/mcguirepr89/BirdNET-Pi/discussions/609#discussioncomment-3919881

After that you'll want to have the command `tail -f /var/log/syslog` running in a terminal. Then, you can repeat the behavior that causes the password to be reprompted over and over, and check to see what errors pop up in the terminal.

**Comment by mcguirepr89** [comment]
If I were to take a guess, I would guess that the installation actually failed because it is likely that the `birder` user is not in `/etc/sudoers.d/` with a NOPASSWD entry. @joshindc, can you confirm whether you've provided passwordless sudo permissions for the `birder` user?

This is what needs to exist to have your vanilla debian take on some Raspi-isms: File path = `/etc/sudoers.d/010_pi-nopasswd`

```
birder ALL=(ALL) NOPASSWD: ALL
```

If that `sudoers.d` file does not exist, you'll need to create it, then uninstall BirdNET-Pi and reinstall it.
